src/inc/til/some.h

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include <array>
+
+namespace til // Terminal Implementation Library. Also: "Today I Learned"
+{
+    template<class _Ty, size_t _Size>
+    class some : std::array<_Ty, _Size>
+    {
+    public:
+        using value_type = _Ty;
+        using size_type = size_t;
+        using difference_type = ptrdiff_t;
+        using pointer = _Ty*;
+        using const_pointer = const _Ty*;
+        using reference = _Ty&;
+        using const_reference = const _Ty&;
+
+        using iterator = _STD _Array_iterator<_Ty, _Size>;

```

Comment:
nah, we can't do this. `_STD` is super private, as are `_UnderscoreUppercase` bros.

This can be achieved with `using array::iterator;` (when deriving) or by deriving the type using `decltype` and `declval` (when composing)

Reply:
OK.